routes/volunteer_hours/volunteer_hours.js

```diff
@@ -17,4 +18,36 @@ hoursRouter.get('/unapproved', async (req, res) => {
   }
 });
 
+// Get hours by volunteer
+hoursRouter.post('/create', async (req, res) => {
+  console.log(req.body);
```

Comment:
⚠️ **[eslint]** <[no-console](https://eslint.org/docs/rules/no-console)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected console statement.

Reply:
pls remove this log statement